sources/Schedules/Program.cs

```diff
@@ -0,0 +1,17 @@
+using Schedules.Services;
```

Comment:
@kyranet how do we feel about top level Program.cs? Should I move this over to the older method; as with the rest of the services; or should we move the other services to using top level Program.cs?

Reply:
Please do so for all the other services, this is godlike.

Reply:
noted, that can be done easily in a seperate PR (: